Makefile

```diff
@@ -115,7 +115,7 @@ publish: prep-install-scripts
 	fi
 	az storage blob upload-batch -d porter/$(PERMALINK) -s bin/$(VERSION)
 
-install: build
+install:
```

Comment:
Should this be build-porter then instead of dropping the requirement completely? Either way please update the CONTRIBUTING.md file to let people know how to run `make install` properly as that target is documented in there.

Reply:
I've updated the CONTRIBUTING doc.  I'd still vote for decoupling `install` with any of the `build*` targets -- what do you think?